keyspace.go

```diff
@@ -121,6 +121,39 @@ func (k *k) MultiTimeSeriesTable(name, indexField, timeField, idField string, bu
 	}
 }
 
+func (k *k) FlakeSeriesTable(name string, bucketSize time.Duration, row interface{}) FlakeSeriesTable {
+	m, ok := toMap(row)
+	if !ok {
+		panic("Unrecognized row type")
+	}
+	m[flakeTimestampFieldName] = time.Now()
+	m[bucketFieldName] = time.Now()
+	return &flakeSeriesT{
+		Table: k.NewTable(fmt.Sprintf("%s_flakeSeries", name), row, m, Keys{
+			PartitionKeys:     []string{bucketFieldName},
+			ClusteringColumns: []string{flakeTimestampFieldName, "Id"},
+		}),
+		bucketSize: bucketSize,
+	}
+}
+
+func (k *k) MultiFlakeSeriesTable(name, indexField string, bucketSize time.Duration, row interface{}) MultiFlakeSeriesTable {
+	m, ok := toMap(row)
+	if !ok {
+		panic("Unrecognized row type")
+	}
+	m[flakeTimestampFieldName] = time.Now()
+	m[bucketFieldName] = time.Now()
+	return &multiFlakeSeriesT{
+		Table: k.NewTable(fmt.Sprintf("%s_by_%s_flakeSeries", name, indexField), row, m, Keys{
```

Comment:
I think this name should be consistent with how Gocassa names other types of tables. So this would become `<name>_multiFlakeSeries_<indexField>_<idField>` (assuming you make the other change to allow the Id field to be configurable).